toptal/picasso
# feat: set default tooltip delay to .5 seconds

[SPC-379]

### Description

Delay showing and hiding tooltips to mimic the default `title` attribute experience. See Slack discussion https://toptal-core.slack.com/archives/CM8KE183V/p1591790386361600.

### How to test

- Hover elements with tooltip and observe a .5 second delay to show the tooltip
- Quickly hover over and out of elements with a tooltip, and observe that the tooltip does NOT show up (default Material UI behavior, that mimics the default `title` attribute behavior), which is exactly what we want
- I also observed that if you hover over an element, in sub-sequent hovers the tooltip shows up immediately - again, this is also the default browser `title` attribute behavior.
- Note that I added both enter and leave delay - let me know if leave delay also makes sense, otherwise, we can easily change it accordingly (remove, make the "leave" delay shorter or even longer).
- Note that the screenshot tests for `TypographyOverflow` no longer display a tooltip, as you can see in the temploy. The tooltip still works, but because now it has a delay, it's no longer captured in the screenshot.

[SPC-379]: https://toptal-core.atlassian.net/browse/SPC-379
